### DISCUSSION TITLE
Installation error with v0.64.x

### DISCUSSION THREAD

**Discussion by charley5cn** [outline/outline#3589] [General] [answered]
I used v0.63.0 and it works very well. Today I am going to install v0.64.2 from source, and the values of `.env` file are what I used in v0.63.0. But when I install the v0.64.2 follow the document that outline provided, the terminal display this:
【However, when I install v0.63.0, it works】
<img width="955" alt="image" src="https://user-images.githubusercontent.com/51190452/170423101-9ab9d040-aaa3-4589-b564-cb6213e33116.png">

**Comment by charley5cn** [comment]
Nginx is used for proxy. It shows that redirected too many times when I visit my website.

**Reply by tommoor** [reply]
No clue I'm afraid. Can you use a terminal that supports the color characters? It's impossible to read logs that look like this…

**Reply by charley5cn** [reply]
Hmmm...Sorry, I am a novice of deploying outline. The output in the screenshot is that I use the command `yarn start`.
It seems different to v0.63.0 when after running `yarn start` like this.
<img width="752" alt="image" src="https://user-images.githubusercontent.com/51190452/170438763-8dabef7c-4eb8-4655-8a1e-19ee59475b15.png">

The method I install v0.64.2 from source just like v0.63.0 follow the document.

**Reply by tommoor** [reply]
You're running in development mode. Use `NODE_ENV=production`. Also `v0.64.3` is latest now.

**Reply by charley5cn** [reply]
hmmm...I deployed the v0.64.3 just now, and used command `set NODE_ENV=production`, but when I run the `yarn start`, the output is still like the first screenshot. 😭 😭 😭

**Reply by almereyda** [reply]
I am having a difficulty in understanding what the perceived error is here.

Is it, that the colour coding works with 0.63.0, but not with 0.64.3, or maybe that there is no visual discrimination in the log output about the `NODE_ENV` in which Outline was started? Ghost for example gives you a line on the beginning of its log for that purpose.

From what I can depict from the two images, there is a difference in the logging behaviour of Outline, which seems to have switched to ND-JSON for easier log aggregation and structured querying. Eventually some regression appeared here, where the coloured log output has not been sanitized before passing into the JSON dictionary, why ANSI colour codes are escaped directly into the JSON string.

Which qualifies this discussion to be escalated into an issue.

**Reply by charley5cn** [reply]
You're right, but my problem is:
<img width="432" alt="image" src="https://user-images.githubusercontent.com/51190452/170646559-d8e83643-0c71-4d33-9010-8d160960b1a2.png">
I don't know what happend by reading the log, as you see in figure 1, the encoding may be is set in a wrong way (I am not sure), the `message` is not readable for me.

**Comment by charley5cn** [comment, marked as answer]
I read the document again, and I noticed that the option `FORCE_HTTPS` should be set into `false` in `.env` file, because I have set 301 redirect in my nginx `conf` file.

**Comment by charley5cn** [comment]
I found that the new version loads faster than before, nice!
